### ISSUE TITLE
Rule: in line comments

### ISSUE THREAD

**Issue by bhirsz** [MarketSquare/robotframework-robocop#84]
```
Test
  Keyword1
  # Keyword2
  Keyword3
```
If someone is commenting out the whole lines, it should report warning

**Comment by wagnerd** [comment]
Hey, :)
Saw your talks on the RoboCon and thought I want to try to participate a bit while getting familiar with the tool :)
Just opened a PR for this (My first on GitHub! :D).

**Comment by mnojek** [comment]
Hey, @wagnerd !
I'm glad that you liked our talks. Thanks for the contribution! 
This is a really old issue and I think we need to reconsider it again. 
Should this actually be reported? I use comments like this from time to time and I would not consider them a warning. 
What do you think, @bhirsz ?

**Comment by bhirsz** [comment]
The PR looks well done - good to see you manage to figure out how to make tests etc :). 

And I also think it's very old issue and we would need to reconsider the purpose of it. Most of the people don't think of such comments as an issue. We would need to find the specific example where it would be considered as the code smell (ie block comments in some sections etc )

**Comment by szabolcstoth** [comment]
> Should this actually be reported?

I think it really depends on the comment itself. :)

It can happen that the comment basically just repeats what the developer can figure out from the used keyword's name, these kind of comments are not necessary. And there are cases when certain steps need to be explained a little further.

Usually in our project we use line comments only for # TODO.

**Comment by wagnerd** [comment]
Are you at the RoboCon sprints today? Maybe we can have a chat there about a possible topic which is still wanted? :)

**Comment by bhirsz** [comment]
I will try to join sprints in next 2 hours (12.30 GMT +1). But to sum up current plans:

1) We want to integrate our tools with various IDEs - I'll working with LSP guys to improve our core methods so it can be reused by them. 
2) Get context of loaded files (so embedded keywords, or rules like "this keyword is defined but not used" are possible
3) Various rules like: #209, #185, #184, #112. Some of them should be easier to implement than other stuff

**Comment by mnojek** [comment]
I think we decided to close this ticker, right @bhirsz ?

**Comment by bhirsz** [comment]
Yes, I will close this issue and related PR - although if there is need for such rule, you can always create external rule and run it with robocop with:
```
robocop --ext-rules your_rule.py
```
